### PR TITLE
add fullscreen icon

### PR DESCRIPTION
**Pull request by jancborchardt**
Useful for Gallery, Spreed and other things. We should also backport it to the stable branches. cc @karlitschek 

Please review @nextcloud/designers

### CODE REVIEW

**Comment by LukasReschke** [comment]
LGTM! 🚀

**Comment by skjnldsv** [comment]
To the moon!
👍

File: core/img/actions/fullscreen.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" height="16" width="16" version="1.1">
```
Comment (nickvergessen) [inline core/img/actions/fullscreen.svg]: missing viewbox, so firefox does not rage quit?

**Comment by karlitschek** [comment]
great idea :+1:

**Comment by rullzer** [comment]
Why backport? This is introducing something new.

**Comment by MariusBluem** [comment]
Because Spreed, Gallery and maybe other apps will depend on it 😁 @rullzer

**Comment by jancborchardt** [comment]
Yup, what @MariusBluem said. If we use it in these apps, they need to stay backwards-compatible.

**Comment by rullzer** [comment]
Well they should not depend on stuff that wasn't there. Ship their own icons. Or say they only support NC11.

But I'm not having the whole backport discussion again.

**Comment by MorrisJobke** [comment]
:+1:

**Comment by jancborchardt** [comment]
stable10 #1818, stable9 #1819

**Pull request merged**